packages/opentelemetry-core/src/metrics/NoopMeter.ts

```diff
@@ -70,9 +71,9 @@ export class NoopMetric<T> implements Metric<T> {
    * Returns a Handle associated with specified label values.
    * It is recommended to keep a reference to the Handle instead of always
    * calling this method for every operations.
-   * @param labelValues the list of label values.
+   * @param labels the object of label set.
```

Comment:
```suggestion
   * @param labels the object of label set.
```
* @param labels the `LabelSet` used to associate with this metric handle.